components/domains/renewal.tsx

```diff
@@ -51,8 +60,12 @@ const Renewal: FunctionComponent<RenewalProps> = ({ groups }) => {
   const [salesTaxRate, setSalesTaxRate] = useState<number>(0);
   const [salesTaxAmount, setSalesTaxAmount] = useState<string>("0");
   const [callData, setCallData] = useState<Call[]>([]);
-  const [price, setPrice] = useState<string>("");
-  const [balance, setBalance] = useState<string>("");
+  const [priceInEth, setPriceInEth] = useState<string>(""); // price in ETH
+  const [price, setPrice] = useState<string>(""); // price in displayedCurrency, set to priceInEth on first load as ETH is the default currency
```

Comment:
It's not set to priceInEth atm

Reply:
Ah yes I updated the comment, in the case of the renewal page we compute the price depending on the number of domains that are selected on the first load so no need to initialize it.